### ISSUE TITLE
Form cannot store field vaules in diffente "free" storage tables

### ISSUE THREAD

**Issue by michaelhq** [Octopoos/SEBLOD#200]
The issue was already reported two years ago:
[http://www.seblod.com/community/forums/forms-content-types/fields-storage](url)
[http://www.seblod.com/resources/tracker/13284](url)

Because of a limitation of the installation of MySQL at my web space provider, I cannot store more than 10 fields with datatype "TEXT" in one table (e.g. for WYSIWYG Fields). When I create more than 10 fields ind phpMyAdmin and try to store some content. The following error code appears: "MySQL: Error Code: 1118 Row size too large (> 8126). Changing some columns to TEXT or BLOB"

To workaround this error I set up use two tables to store the content. The fields up to ten were set with storage free to the first table, fields up to 20 were set with storage free to the second table (what is btw no problem in the set up of the construction of the form).

When using the form, the content of the field which are set to storage "free" with the first table are saved, the filed wihch are set to the second table are not saved by the admin form. If I edit some content in the second table directly in e.g. phpMyAdmin, the content is loaded correctly by the admin form.

**Comment by klas** [comment]
This is not an issue, Seblod is designed to store to single table. I would advise you to change hosting provider.

**Comment by michaelhq** [comment]
I agree, content should be stored in not more than one additional table (e.g. #__content and #__cck_store_form_XYZ).

But therefore it would be nice, if Seblod would not be able to load content in forms from multiple additional "free" tables (which then cannot be stored after editing). The option "free" under storage should be limited in a form to one additional table (e.g. display a warning).

Because moving to another hosting provider is not an option for me, I solved the problem now with JSON fields. I store now the content of 12 WYSISWG fields (JSON) in one field of a table in the MYSQL database. So i do not need any "free" storage anymore. The content is now saved in the two standard tables (e.g. #__content and #__cck_store_form_XYZ).

**Comment by klas** [comment]
Feature request
